src/accelerate/accelerator.py

```diff
@@ -324,10 +324,13 @@ def __init__(
 
         if is_ipex_available():
             if ipex_plugin is None:  # init from env variables
-                ipex_plugin = IntelPyTorchExtensionPlugin()
+                ipex_plugin = (
+                    IntelPyTorchExtensionPlugin() if os.environ.get("IPEX_ENABLED", "false") == "true" else None
+                )
```

Comment:
Considering ipex is a device and not really a plugin like megatron, we want it to create the plugin to automatically pick up on using xpu. cc @sgugger to verify.

Reply:
yes, by default it should be used if available unless the user explicitly said no.

Reply:
agree, I change logic like ipex is used unless the user explicitly said no by export IPEX_ENABLED or set it in accelerate config